src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -150,7 +151,7 @@ export default class FilingHistoryList extends Mixins(FilingMixin) {
 
       // build correction filing
       let correctionFiling: CorrectionFilingIF
-      if (this.isFirm || this.isBenBcCccUlc) {
+      if (this.isFirm || this.isBenBcCccUlc || this.isCoop) {
```

Comment:
I checked with our BA (Mihai) and PO (Dave McK) and they said we will have other entity types later, so this code is _perfect_ -- it will not accidentally allow new entity types.
👍